source/simulator/stokes_matrix_free.cc

```diff
@@ -2063,8 +2063,14 @@ namespace aspect
           dealii::LinearAlgebra::distributed::BlockVector<double> tmp_dst = solution_copy;
           dealii::LinearAlgebra::distributed::BlockVector<double> tmp_src = rhs_copy;
           time_this("stokes_vmult", 10,
-                    [&] {stokes_matrix.vmult(tmp_dst, tmp_src);},
-                    [&] {tmp_src = tmp_dst;}
+                    [&] ()
```

Comment:
Astyle indentation is a bit ugly. Maybe we need to reconsider clang format some day...

Reply:
Or a newer version of astyle that understands C++11 and later...